### PR TITLE
Add missing Slovak translation

### LINKED ISSUE #13324: Missing Slovak translation
### Description

SK translation of **federatedIdentityUnavailableMessage** is missing in **themes/src/main/resources-community/theme/base/login/messages/messages_sk.properties**

The translation should be following: Používateľ {0} overený poskytovateľom identity {1} neexistuje. Kontaktujte svojho správcu.

### PR DESCRIPTION
**Pull request by diaboy96**
Closes #13324

### CODE REVIEW

File: themes/src/main/resources-community/theme/base/login/messages/messages_sk.properties [hunk 1]
```diff
@@ -164,6 +164,7 @@ usernameExistsMessage=Užívateľské meno už existuje.
 emailExistsMessage=E-mail už existuje.
 
 federatedIdentityExistsMessage=Používateľ s {0} {1} už existuje. Ak chcete prepojiť účet, prihláste sa na správu účtov.
+federatedIdentityUnavailableMessage=Používateľ {0} overený poskytovateľom identity {1} neexistuje. Kontaktujte svojho správcu.
```
Comment (martin-kanis) [inline themes/src/main/resources-community/theme/base/login/messages/messages_sk.properties:167]:
```suggestion
federatedIdentityUnavailableMessage=Používateľ {0} autentifikovaný poskytovateľom identity {1} neexistuje. Kontaktujte svojho administrátora.
```

**Review by martin-kanis — CHANGES_REQUESTED** [review]
@diaboy96 Thank you for the PR. For consistency reasons with the rest of the translation, I think, it would be better to use adjective "autentifikovaný" as noun "autentifikácia" is often used in the rest and also noun "administrátora".

**Review by martin-kanis — DISMISSED** [review]
Thanks @diaboy96.

**Review by hmlnarik — CHANGES_REQUESTED** [review]
@diaboy96 Could you please squash the two commits into a single one?

**Comment by diaboy96** [comment]
> @diaboy96 Could you please squash the two commits into a single one?

Of course

**Review by hmlnarik — APPROVED** [review]
Thank you!

**Pull request merged**